src/Model.php

```diff
@@ -108,6 +110,16 @@ public function resetValidator()
         $this->validator = null;
     }
 
+    public function removeExtraAttributes()
+    {
+        $columns = Schema::getColumnListing($this->getTable());
```

Comment:
Actually, yes. Because there is no other way to get original attributes from the Model. Tried others approach with native Models methods, but no lucky.

Reply:
We could have a method `$model->getFieldNames()`. That will skip the query to the database and improve performance.